extensions/sql-bindings/src/common/constants.ts

```diff
@@ -69,11 +69,11 @@ export const connectionProfile = localize('connectionProfile', 'Select a connect
 export const userConnectionString = localize('userConnectionString', 'Enter connection string');
 export const selectConnectionString = localize('selectConnectionString', 'Select SQL connection string method');
 export const includePassword = localize('includePassword', 'Do you want to include the password from this connection in your local.settings.json file?');
-export const enterPasswordPrompt = localize('enterPasswordPrompt', 'Enter the password to be used for the connection string');
-export const enterPasswordManually = localize('enterPasswordManually', 'Enter password or press escape to cancel');
 export const userPasswordLater = localize('userPasswordLater', 'In order to user the SQL connection string later you will need to manually enter the password in your local.settings.json file.');
 export const openFile = localize('openFile', "Open File");
 export const closeButton = localize('closeButton', "Close");
+export const enterPasswordPrompt = localize('enterPasswordPrompt', 'Enter the password to be used for the connection string that will be saved in the local.settings.json file');
```

Comment:
@dzsquared can you help with a string that's a bit less verbose?

Reply:
![image](https://user-images.githubusercontent.com/23587151/168943780-84494153-d03b-4a0b-bfd9-de84062c2bcb.png)

Reply:
I think this would benefit from a flowchart diagram to see how these prompts appear - the context of previous prompts matters a lot here so that would help decide how these should be displayed to the user

Reply:
I can try and create a diagram but I may not have much time to do this. For now, I will do the refactoring you suggested below and how I started the scenarios. If I get the Figma done I will be sure to update this PR with the link.

Reply:
Maybe simplify to `Enter connection password to save in local.settings.json`? 

Although it might be better to start off with a quickpick prompt 

`Enter password to save in local.settings.json? If not then you will need to enter it manually later.` - then you can get rid of the warning for this path. It may be better to do that for all the paths honestly - right now we don't really tell the user why they should/shouldn't save the password until after they've chosen which isn't great.

Reply:
Adding another quickpick may introduce more risk at this point, even if it may make it more clearer. For now just make it clearer as you suggested.